.github/workflows/api.yml

```diff
@@ -41,6 +41,7 @@ jobs:
           path: seqan3/submodules/seqan
 
       - name: Configure APT
+        continue-on-error: true
```

Comment:
Maybe put it into the script itself? Otherwise, I like that you can see here that the apt failed.

Reply:
There is either fail or no fail in a step, I cannot skip on failure. So it doesn't matter where I put it. This way, it's at least not hidden inside the script that we do not fail on error